config/monica.php

```diff
@@ -255,6 +255,16 @@
     */
     'darksky_api_key' => env('DARKSKY_API_KEY', null),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Configure default rate limit for route services per minute
+    |--------------------------------------------------------------------------
+    |
+    | Configure rate limit for route services per minute
+    */
+    'rate_limit_api' => env('DEFAULT_RATE_LIMIT_PER_MINUTE_API', 60),
+    'rate_limit_oauth' => env('DEFAULT_RATE_LIMIT_PER_MINUTE_OAUTH', 5),
```

Comment:
Instead of `` DEFAULT_RATE_LIMIT_PER_MINUTE_API ``  and `` DEFAULT_RATE_LIMIT_PER_MINUTE_OAUTH ``  wouldn't it be better to just say  `` RATE_LIMIT_PER_MINUTE_API ``  and `` RATE_LIMIT_PER_MINUTE_OAUTH`` ? The default one are the ``60`` and ``5`` values.

Reply:
Thank your for your request 👍 
I've added the requested changes.